cmd/kusk/cmd/root.go

```diff
@@ -49,7 +49,9 @@ var rootCmd = &cobra.Command{
 	Long:  ``,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		analytics.SendAnonymousCMDInfo(nil)
-		if cmd.Name() != generateCmd.Name() && build.Version != "latest" {
+		if cmd.Name() != generateCmd.Name() &&
+			cmd.Name() != ipCmd.Name() &&
```

Comment:
Hmm, this could blow up, like for example when you're piping the command to something else. 

I think a better solution that works always is to check if the output is an interactive terminal or not, and don't print if it is not.

https://github.com/mattn/go-isatty
cc @mbana